.github/labeler.yml

```diff
@@ -1,3 +1,15 @@
+documentation:
+  - AUTHORS
+  - CONTRIBUTING.rst
```

Comment:
What about something like [`**/*.+(rst|md)`](https://globster.xyz/?q=**%2F*.%2B(rst%7Cmd)&f=%2Fmyapp%2Freadme.md%2C%2Fmyapp%2Fconfig%2Fstaging.js%2C%2Fmyapp%2Fconfig%2Fproduction.js%2C%2Fmyapp%2Fsrc%2Fservices%2Futils.js%2C%2Fmyapp%2Fsrc%2Fservices%2Ftimezone.ts%2C%2Fmyapp%2Fsrc%2Fcontrollers%2Fhealth.js%2C%2Fmyapp%2Fsrc%2Fcontrollers%2Fuser.module.ts%2C%2Fmyapp%2Fassets%2Flogo.png%2C%2Fmyapp%2Fassets%2Flogo_small.png%2C%2Fmyapp%2Fassets%2Ffavicon.ico%2C%2Fasd%2Fasd%2Ftext.rst&embed=1")?

That would catch any piece of documentation deeply buried in the repo and perhaps Doc folk/s would want to know about and help on those too?

Reply:
Good idea. Updated!